Content/.template.config/template.json

```diff
@@ -2,25 +2,28 @@
     "$schema": "http://json.schemastore.org/template",
     "author": "Jimmy Byrd",
     "classifications": [
-        "Scaffold"
+        "Scaffold", "Console", "Library"
     ],
+    "description": "A full-featured template for creating console or library solutions with documentation and release workflows.",
```

Comment:
These descriptions give nice tooltips, but they also help in the CLI tooltips/help as well.